### PR TITLE
Remove unnecessary code from identity auth test

### PR DESCRIPTION
**Pull request by jsmolar**

### CODE REVIEW

**Review by pehala — APPROVED** [review]

File: testsuite/tests/kuadrant/authorino/identity/auth/test_auth_identity.py [hunk 1]
```diff
@@ -22,13 +22,6 @@ def oidc_provider(request) -> OIDCProvider:
     return request.getfixturevalue(request.param)
 
 
-# pylint: disable=unused-argument
-@pytest.fixture(scope="module")
-def authorization_name(blame, oidc_provider):
-    """Ensure for every oidc_provider we have a unique authorization"""
-    return blame("authz")
```
[the remark this answers is not in the document]
Comment (jsmolar) [reply testsuite/tests/kuadrant/authorino/identity/auth/test_auth_identity.py:29]: we already have `authorization_name` in another conftest. I don't understand how this simple blame is doing something more.

[the remark this answers is not in the document]
Comment (jsmolar) [reply testsuite/tests/kuadrant/authorino/identity/auth/test_auth_identity.py:29]: I may be wrong here. If you know how is this fixture beneficial, please enlighten

[the remark this answers is not in the document]
Comment (averevki) [reply testsuite/tests/kuadrant/authorino/identity/auth/test_auth_identity.py:29]: iirc this fixture with the `oidc_provider` fixture as a dependency ensures that for each oidc provider will be created individual AuthConfig with a different name (without it, same name and same python object(?) will be used)

[the remark this answers is not in the document]
Comment (averevki) [reply testsuite/tests/kuadrant/authorino/identity/auth/test_auth_identity.py:29]: Was added here: https://github.com/Kuadrant/testsuite/pull/159

**Review by averevki — APPROVED** [review]

**Pull request merged**